### ISSUE TITLE
Mako Error

### ISSUE THREAD

**Issue by vlsoft** [SickRage/SickRage#1729]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: Master
OS: Linux

What you did: update  

What happened:

File /volume1/.@plugins/AppCentral/sickbeard-tvrage/SickBeard-TVRage/gui/slick/views/home.mako:295, in render_content:
                                            <%
KeyError: 300472

What you expected: Show list

Logs:
```
PASTE LOGS HERE
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by jeremybdk** [comment]
I do have the same error, here is the debug log link to https://gist.github.com/jeremybdk/447330f45a237c9db8c14ea45711c825

Running on Windows with latest commit.

List of show is not working and the error is the same as above, except key error is 234851

Anything else you might need ? How can I help you debug ?

**Comment by neoatomic** [comment]
All is working fine here, so you will have to provide some more info about your setup etc.
What resolution,? accessing with a mobile device.? Only getting the error on the Home page.? what Browser.? ...

Also see if you can shutdown sickrage and remove everything in the cache folder (except images) and restart SR.

**Comment by jeremybdk** [comment]
So resolution is 1920x1080 using chrome on a desktop computer with Windows 10. Only the index seems to be broken.  I tried to clear the cache folder but still having the issue. And I'm using the list view.

**Comment by Arcanemagus** [comment]
I'm seeing the exact same thing as @jeremybdk, but with SR running in a FreeBSD jail as the only difference in situation. I was seeing this intermittently on I believe the schedule page as well, but I can't reproduce that any more.

**Comment by iearmand** [comment]
I'm seeing the same thing on the Show list page. Everything else seems to working I'm on OSx.
Also get this error on the page
"File /Applications/SickRage/gui/slick/views/home.mako:295, in render_content:
                                            <%
KeyError: 300472"

**Comment by miigotu** [comment]
hotfixing now.

@beschoenen I blame you xDD

**Comment by beschoenen** [comment]
Well deserved 😞

**Comment by iearmand** [comment]
Thanks for the quick fix gents.

**Comment by jeremybdk** [comment]
Thanks for the fix and for improving SickRage

**Comment by netrbrunet** [comment]
I've recently updated sickrage, and am getting a very similar error:

`
File /home/<user>/.sickrage/gui/slick/views/home.mako:295, in render_content:
                                            <%
IndexError: Index must be int or string
`

I was having a hard time seeing anyone else with the error, and finally stumbled upon this recent thread. I updated again just 5 minutes ago, but I still have the same issue (and sickrage restarted as part of the update). I'm at a real loss here. I even deleted my db's to ensure there was no problem during the first update. As soon as I added a single show back (via add existing show (in folder)), the error started again.

OS: Ubuntu 14.04
res: 1920 x 1080 (on my tv)
I'm not sure what the view style is set to - but it was the show list that had banners, one per line.
